include/ctarget/set.h

```diff
@@ -211,3 +211,8 @@ do { \
 #endif // MODAL_REACTORS
 
 #endif // CTARGET_SET
+
+// FIXME: Temporary workaround to enable LET reactions and changing lf_schedule and lf_time API
```

Comment:
Do we need to treat this as a temporary change?  Couldn't it be permanent?

Reply:
I guess it could be permanent.